src/core/shape_inference/include/scaled_dot_product_attention_shape_inference.hpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#pragma once
+
+#include "openvino/op/scaled_dot_product_attention.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace op {
+namespace v13 {
+template <class T, class TRShape = result_shape_t<T>>
+std::vector<TRShape> shape_infer(const ScaledDotProductAttention* op,
+                                 const std::vector<T>& input_shapes,
+                                 const bool& iscausal) {
```

Comment:
```suggestion
                                 const bool& iscausal) {
```
Remove this parameter, it will not fit for common signature of this function

Reply:
Done